huntd-graphql-serverless/src/graphql/generated.serverless.ts

```diff
@@ -0,0 +1,1023 @@
+import { GraphQLClient } from 'graphql-request';
+import { print } from 'graphql';
+import { GraphQLError } from 'graphql-request/dist/types';
+import gql from 'graphql-tag';
+export type Maybe<T> = T | null;
+export type Exact<T extends { [key: string]: unknown }> = { [K in keyof T]: T[K] };
+/** All built-in and custom scalars, mapped to their actual values */
+export type Scalars = {
+  ID: string;
+  String: string;
+  Boolean: boolean;
+  Int: number;
+  Float: number;
+  GraphQLDateTime: string;
+  Upload: any;
+};
+
+export type Subscription = {
+  __typename?: 'Subscription';
+  candidateProfileStatusUpdated: CandidateProfile;
+  newMessage?: Maybe<ChatMessage>;
+  profileConnectionUpdated: ProfileConnection;
+  recruiterProfileStatusUpdated: RecruiterProfile;
+  userUnreadMessagesCountUpdated?: Maybe<User>;
+};
+
+export type Query = {
+  __typename?: 'Query';
+  adminUser?: Maybe<User>;
+  authUser?: Maybe<User>;
+  candidateProfileBySlug?: Maybe<CandidateProfile>;
+  candidateProfilesBySubscription: Array<CandidateProfile>;
+  employmentLocations?: Maybe<Array<EmploymentLocation>>;
+  employmentTypes?: Maybe<Array<EmploymentType>>;
+  englishLevels?: Maybe<Array<EnglishLevel>>;
+  jobExperiences?: Maybe<Array<JobExperience>>;
+  latestCandidateProfile?: Maybe<CandidateProfile>;
+  latestRecruiterProfile?: Maybe<RecruiterProfile>;
+  profileConnection?: Maybe<ProfileConnection>;
+  publicCandidateProfiles: Array<CandidateProfile>;
+  publicRecruiterProfiles: Array<RecruiterProfile>;
+  recruiterProfileBySlug?: Maybe<RecruiterProfile>;
+  serviceUser?: Maybe<User>;
+  specializations?: Maybe<Array<Specialization>>;
+  technologies?: Maybe<Array<Technology>>;
+  usersOAuthProviders?: Maybe<Array<OAuthToken>>;
+  usersSearchSubscriptions?: Maybe<Array<UsersSearchSubscription>>;
+};
+
+
+export type QueryCandidateProfileBySlugArgs = {
+  slug: Scalars['String'];
+};
+
+
+export type QueryCandidateProfilesBySubscriptionArgs = {
+  subscriptionLastInteract: Scalars['GraphQLDateTime'];
+  where: PublicProfilesParameters;
+};
+
+
+export type QueryProfileConnectionArgs = {
+  id: Scalars['Int'];
+};
+
+
+export type QueryPublicCandidateProfilesArgs = {
+  where?: Maybe<PublicProfilesParameters>;
+};
+
+
+export type QueryRecruiterProfileBySlugArgs = {
+  slug: Scalars['String'];
+};
+
+
+export type QuerySpecializationsArgs = {
+  query?: Maybe<Scalars['String']>;
+};
+
+
+export type QueryTechnologiesArgs = {
+  query?: Maybe<Scalars['String']>;
+};
+
+export type Mutation = {
```

Comment:
looks like a lot of queries has been generated
let's put only what we need in lambdas